package_build/ml-pipelines-sdk/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools>=72", "wheel", "pytest-runner", "tomli"]
```

Comment:
Hmm, what do we need `pytest-runner` for as a build-time dependency?

Reply:
>Hmm, what do we need pytest-runner for as a build-time dependency?

Good catch. No it's not required. I missed removing it from project directories.